crates/local-cluster-runner/Cargo.toml

```diff
@@ -16,7 +16,7 @@ workspace-hack = { version = "0.1", path = "../../workspace-hack" }
 restate-core = { workspace = true }
 restate-metadata-server = { workspace = true }
 # nb features here will also affect the compiled restate-server binary in integration tests
-restate-types = { workspace = true, features = ["unsafe-mutable-config"] }
+restate-types = { workspace = true, features = ["unsafe-mutable-config", "test-util"] }
```

Comment:
are you sure this doesn't break restate-server by making it start with random base-dir every time?

Reply:
I didn't know `test-util` has this side effect. I will take it back.